### PR TITLE
Enable to active record 3

### PR DESCRIPTION
**Pull request by lucasprag**
Hi guys, 

I just want to help, so I did some commits to:
- let me be able to use this gem with active record 3.2, I tested and it worked
- change the default rake to be rake test
- add syntax highlight to the readme file on ruby code

Any feedback will be helpful.

Thank you very much. =)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,22 +30,26 @@ Or install it yourself as:
 
 Assuming you've added unique index:
 
-    class AddIndexToThing < ActiveRecord::Migration
-      disable_ddl_transaction!
+```ruby
```
Comment (ebenoist) [inline README.md]: 👍

File: README.md [hunk 2]
```diff
@@ -54,25 +58,28 @@ enter your database.
...
+thing.errors[:somefield] == "somefield has already been taken"
+# => true
+# => raises ActiveRecord::RecordNotUnique
+```
 
 ## Testing
 
 You'll need a database that supports unique constraints.
-This gem has been tested with PostgreSQL and SQLite only.
+This gem has been tested with PostgreSQL, MySQL and SQLite.
```
Comment (ebenoist) [inline README.md]: 👍

File: Rakefile [hunk 3]
```diff
@@ -1,2 +1,10 @@
 require "bundler/gem_tasks"
 
+begin
+  require "rspec/core/rake_task"
+  RSpec::Core::RakeTask.new(:spec)
+
+  task default: :spec
+rescue LoadError
+end
+
```
Comment (ebenoist) [inline Rakefile]: Can you kill the extra whitespace here?

File: README.md [hunk 4]
```diff
@@ -54,25 +58,28 @@ enter your database.
...
 
 ## Contributing
 
-1. Fork it ( https://github.com/[my-github-username]/rescue_unique_constraint/fork )
+1. [Fork it](https://github.com/reverbdotcom/rescue-unique-constraint/fork)
```
Comment (ebenoist) [inline README.md]: 👍

**Comment by ebenoist** [comment]
Everything here looks good to me!

File: Rakefile (hunk 3 above)
Comment (lucasprag) [reply Rakefile]: Sure =) 8cea7992236c20f959de994ed32621d69b26e66f

**Pull request merged**